third_party/terraform/tests/resource_storage_transfer_job_test.go

```diff
@@ -0,0 +1,185 @@
+					resource.TestCheckResourceAttr("google_storage_transfer_job.transfer_job", "schedule.0.schedule_end_date.0.month", "10"),
+					resource.TestCheckResourceAttr("google_storage_transfer_job.transfer_job", "schedule.0.schedule_end_date.0.day", "1"),
+					resource.TestCheckResourceAttr("google_storage_transfer_job.transfer_job", "schedule.0.start_time_of_day.0.hours", "0"),
+					resource.TestCheckResourceAttr("google_storage_transfer_job.transfer_job", "schedule.0.start_time_of_day.0.minutes", "30"),
+					resource.TestCheckResourceAttr("google_storage_transfer_job.transfer_job", "schedule.0.start_time_of_day.0.seconds", "0"),
+					resource.TestCheckResourceAttr("google_storage_transfer_job.transfer_job", "schedule.0.start_time_of_day.0.nanos", "0"),
+				),
+			},
+		},
+	})
+}
+
+func testAccStorageTransferJob_basic(project string) string {
+	return fmt.Sprintf(`
+data "google_storage_transfer_project_service_account" "default" {
+  project       = "%s"
+}
+
+resource "google_storage_bucket" "data_source" {
+  name          = "test-data-source-bucket-%s"
+  project       = "%s"
+  force_destroy = true
+}
+
+resource "google_storage_bucket_iam_member" "data_source" {
+  bucket        = "${google_storage_bucket.data_source.name}"
+  role          = "roles/storage.admin"
+  member        = "serviceAccount:${data.google_storage_transfer_project_service_account.default.email}"
+
+  depends_on    = [
+    "google_storage_bucket.data_source",
+    "data.google_storage_transfer_project_service_account.default"
+  ]
+}
+
+resource "google_storage_bucket" "data_sink" {
+  name          = "test-data-sink-bucket-%s"
+  project       = "%s"
+  force_destroy = true
+}
+
+resource "google_storage_bucket_iam_member" "data_sink" {
+  bucket        = "${google_storage_bucket.data_sink.name}"
+  role          = "roles/storage.admin"
+  member        = "serviceAccount:${data.google_storage_transfer_project_service_account.default.email}"
+
+  depends_on    = [
+    "google_storage_bucket.data_sink",
+    "data.google_storage_transfer_project_service_account.default"
+  ]
+}
+
+resource "google_storage_transfer_job" "transfer_job" {
+	description	= "transfer-job-%s"
+	project     = "%s"
+
+	transfer_spec {
+		gcs_data_source {
+			bucket_name = "${google_storage_bucket.data_source.name}"
+		}
+		gcs_data_sink {
+			bucket_name = "${google_storage_bucket.data_sink.name}"
+		}
+	}
+
+	schedule {
+		schedule_start_date {
+			year	= 2018
+			month	= 10
+			day		= 1
+		}
+		schedule_end_date {
+			year	= 2019
+			month	= 10
+			day		= 1
+		}
+		start_time_of_day {
+			hours	= 0
+			minutes	= 30
+			seconds	= 0
+			nanos	= 0
+		}
+	}
+
+	depends_on = [
+		"google_storage_bucket_iam_member.data_source",
+		"google_storage_bucket_iam_member.data_sink",
+	]
+}
+`, project, acctest.RandString(10), project, acctest.RandString(10), project, acctest.RandString(10), project)
+}
+
+func testAccStorageTransferJobExists(n string) resource.TestCheckFunc {
```

Comment:
We don't need `Exists` checks if we're using import-style checks

Reply:
Removed `testAccStorageTransferJobExists`